app/elements/code-editor/language-selector.html

```diff
@@ -1,10 +1,23 @@
+<link rel="import" href="/bower_components/paper-dropdown-menu/paper-dropdown-menu.html">
+<link rel="import" href="/bower_components/paper-menu/paper-menu.html">
+
 <dom-module id="language-selector">
+	<style>
+		paper-dropdown-menu{
+			width: 200px;
+		}
+		option {
+			width: 200px;
+		}
+	</style>
 	<template>
-		<select id="language">
-			<template is="dom-repeat" items="{{languages}}">
-				<option value="{{item.tag}}">{{item.name}}</option>
-			</template>
-		</select>
+		<paper-dropdown-menu id="language" label="Language" selected-item="{{language}}">
+			<paper-menu class="dropdown-content">
+				<template is="dom-repeat" items="{{languages}}">
+					<option value="{{item.tag}}">{{item.name}}</option>
```

Comment:
`paper-item` doesn't have a value attribute that's different from it's label. If i were to use it, i'd have to put a hidden input  inside the paper-item which stores the value. And since visually, they're the same, i prefered plain old `<option>`